exporter/http_test.go

```diff
@@ -238,7 +237,6 @@ func TestSimultaneousMetricsHttpRequests(t *testing.T) {
 
 		os.Getenv("TEST_REDIS5_URI"),
 		os.Getenv("TEST_REDIS6_URI"),
-		os.Getenv("TEST_REDIS_MODULES_URI"),
```

Comment:
From my understanding TEST_REDIS_MODULES_URI is not required here, as modules tested only in `exporter/modules_test.go`

Reply:
Correct. It's more about adding a multitude of redis instances to the pool of instances that get scraped via the `/scrape` endpoint to make sure nothing surprising happens.

Reply:
Cool, thank you for details, reverted this change.